plugin.video.pcloud-video-streaming/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.pcloud-video-streaming" name="PCloud Video Streaming" version="1.2.3" provider-name="Guido Domenici">
+<addon id="plugin.video.pcloud-video-streaming" name="PCloud Video Streaming" version="1.2.5" provider-name="Guido Domenici">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
please consider to set this one to 2.19.0 in the future (so it matches helix). See: http://kodi.wiki/view/addon.xml